packages/web-pkg/src/composables/actions/files/useFileActionsOpenWithApp.ts

```diff
@@ -22,13 +22,13 @@ export const useFileActionsOpenWithApp = ({ appId }: { appId: string }) => {
     dispatchModal({
       elementClass: 'open-with-app-modal',
       title: $gettext('Open file in %{app}', { app: app.name }),
-      hideConfirmButton: true,
       customComponent: FilePickerModal,
+      hideActions: true,
       customComponentAttrs: () => ({
         app,
         parentFolderLink
       }),
-      focusTrapInitial: false
+      focusTrapInitial: 'oc-modal-body-message'
```

Comment:
this should be a css selector I think... so it should be prefixed with a `.` for the class selector

Reply:
fixed, since handled by modal (iFrame foucs)